### DISCUSSION TITLE
Segmentation fault

### DISCUSSION THREAD

**Discussion by forestbat** [opengeos/leafmap#572] [Q&A] [answered]
I want to show map on my jupyter server, but when I try to run code below, it crashed:
```
# leafmap.py
import os
import leafmap
from samgeo import SamGeo, tms_to_geotiff
m = leafmap.Map(center=[37.6412, -122.1353], zoom=2, height="800px")
m
```
and crash only contains these:
```
(test_sam) forestbat@vm-jupyterhub-server:~$ python leafmap.py
Segmentation fault (core dumped)
```
So what probably happened on my server? How to solve it? And I have installed `pygis` before this crash.

**Comment by giswqs** [comment, marked as answer]
Not enough information to determine the cause of the issue. Try creating a fresh conda env to install leafmap.
